### PR TITLE
Add reference datasets

### PR DESCRIPTION
**Pull request by LauraZM9**
Ticket: [[Epic1] Build reference data sets to fill dataset_worker](https://trello.com/c/6AhJgVqL)

Added all the reference datasets to the repo.

### CODE REVIEW

File: ref-data/ASCTraining.csv [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+14,Mental capacity and deprivation of liberty
+15,Moving and handling
+16,Nutrition and hydration
+17,Palliative or end of life care
+18,Physical Disability
+19,Positive Behaviour and support
+20,Safeguarding Adults
+22,"Dignity, Respect, Person Centred care"
+23,"Equality, Diversity and Human Rights training"
+21,Any other not in the above categories
```
Comment (AdamProbert) [inline ref-data/ASCTraining.csv:23]: Either quotes or no quotes  for values

File: ref-data/userType.csv [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+4,Adults with physical disabilities
+7,Adults with sensory impairments
+8,Adults who misuse alcohol or drugs
+31,Adults with an eating disorder
+9,Adults others not in above categories
+45,Any children and young people
+18,Carers of older people
+19,Carers of adults
+20,Carers of children and young people
+21,Other others not in above categories
```
Comment (AdamProbert) [inline ref-data/userType.csv:24]: need a new line at end of file

File: ref-data/jobRole.csv [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+27,Other non-care-providing job roles
+34,Activities worker or co-ordinator
+35,Safeguarding and reviewing officer
+36,Occupational therapist assistant
+37,Nursing Associate
+38,Nursing Assistant
+39,Assessment officer
+40,Care co-ordinator
+41,Care navigator
+42,Any Childrens/young peoples job role
```
Comment (AdamProbert) [inline ref-data/jobRole.csv:35]: end of file new line

File: ref-data/css.csv [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+909,Bristol
+910,North Somerset
+911,South Gloucestershire
+912,Devon
+913,Plymouth
+914,Torbay
+996,Bedford
+997,Central Bedfordshire
+998,Cheshire East
+999,Cheshire West & Chester
```
Comment (AdamProbert) [inline ref-data/css.csv:154]: end of file new line

File: ref-data/ASCTraining.csv [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+31,Mental Health
+32,Autism
+33,Continence Care
+34,Duty of Care
+35,Supervision / Performance management
+36,Stroke
+37,Complaints handling/conflict resolution
+38,Personal Care
+39,Activity provision/Well-being
+40,Sensory disability
```
Comment (AdamProbert) [inline ref-data/ASCTraining.csv:39]: end of file new line

**Review by AdamProbert — CHANGES_REQUESTED** [review]
Looks good, couple of bits and the following request:
- Can you extend the directory name? "reference-data"? , no need to shorten names, we should try and avoid unnecessary cognitive load.

File: ref-data/recruitmentPath.csv [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+211,ACS: Private/Voluntary
+214,Health sector
+216,Other sector
+217,Promotion / Transfer / Development
+219,Not previously employed
+221,Agency
+224,Other sources
+225,
+226,Children's/young people's social care
+227,First role after education
```
Comment (JimmySunDance) [inline ref-data/recruitmentPath.csv:13]: Please remove new line

**Review by JimmySunDance — CHANGES_REQUESTED** [review]
Looks good - but we don't want empty new lines at the end of our csv files.

**Review by AdamProbert — APPROVED** [review]

**Review by JimmySunDance — APPROVED** [review]
LGTM

**Pull request merged**